draft-trammell-taps-post-sockets.md

```diff
@@ -435,14 +436,24 @@ reuse an Association if they can be carried from the same Local to the same
 Remote over the same Paths; this re-use of an Association may implies the
 creation of a new Transient.
 
+Associations may exist without a Message Carrier if required. This may be done if
+peer cryptographic state, e.g., a pre-shared key, is established out-of-band.
+Thus, Associations may be created without the need to send application data
+to a peer, i.e., without a Remote. Associations may also be created from
```

Comment:
I'd prefer to see the asides like e.g. and i.e. written out into "such as a pre-shared key". For the case of not sending application data, do we really mean "without initiating a carrier"?